app/Models/Device.php

```diff
@@ -474,15 +485,7 @@ public function scopeWhereUptime($query, $uptime, $modifier = '<')
 
     public function scopeCanPing(Builder $query)
     {
-        return $query->where('disabled', 0)
-            ->leftJoin('devices_attribs', function (JoinClause $query) {
-                $query->on('devices.device_id', 'devices_attribs.device_id')
-                    ->where('devices_attribs.attrib_type', 'override_icmp_disable');
-            })
-            ->where(function (Builder $query) {
-                $query->whereNull('devices_attribs.attrib_value')
-                    ->orWhere('devices_attribs.attrib_value', '!=', 'true');
-            });
+        return $this->scopeWhereAttributeDisabled($query->where('disabled', 0), 'override_icmp_disable');
```

Comment:
Not really sure what the convention is when using scope functions internally,
but normally you call it with:
```suggestion
        return $query->where('disabled', 0)->whereAttributeDisabled('override_icmp_disable');
```
Maybe thats prettier?